wdl-analysis/src/config.rs

```diff
@@ -119,6 +145,8 @@ struct ConfigInner {
     /// See [`Config::with_fallback_version()`]
     #[serde(default)]
     fallback_version: Option<SupportedVersion>,
+    /// See [`Config::with_ignore_filename()`]
+    ignore: Option<String>,
```

Comment:
Mild preference to make this variable name match the methods.
```suggestion
    ignore_filename: Option<String>,
```